src/schema/contentful/phoenix.js

```diff
@@ -76,6 +76,24 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  type PersonBlock implements Block {
+    "Name of the person displayed on the block"
+    name: String
+    "The type of employer this person is"
+    type: String
+    "Current working status"
+    active: Boolean
+    "Laboral title represented by the person in question"
```

Comment:
I this we can keep this simple with
```suggestion
    "Job title of the person"
```